README.md

```diff
@@ -59,6 +59,23 @@ or [GitHub Actions environment variables](https://docs.github.com/en/actions/lea
 | `ALLURE_RESULTS_DIR`   | Path to the directory where the Allure test report is stored                                                                                                                                                                              | **Yes**  | N/A                                               |
 | `ALLURE_GENERATED_DIR` | Path to the directory that will be used to store the generated report. This directory will be created automatically if it does not exist. ATTENTION - all files that were in this directory before the action was started will be deleted | **No**   | ${GITHUB_WORKSPACE}/morph-allure-generated-report |
 
+### Expiration period environment variables
+The following variables must be passed as 
```

Comment:
`must be passed` and `Required: No` could be confusing. When user does not care bout expiration, is he obliged to pass these vars?

Reply:
> `must be passed` and `Required: No` could be confusing. When user does not care bout expiration, is he obliged to pass these vars?

I don't think it will lead to confusion.
Here I explain where exactly to store values - in secrets or in variables.
To highlight that this is reference information, I have italicized it here and in other paragraphs.

Reply:
"The following variables are _expected to be passed_ via action variables or environment (not secrets)..." --- maybe something like this?

Reply:
> "The following variables are _expected to be passed_ via action variables or environment (not secrets)..." --- maybe something like this?

Fixed